src/components/Sidebar/Cart.js

```diff
@@ -1,7 +1,29 @@
 import React from 'react';
+import Typography from '@material-ui/core/Typography';
+import { withStyles } from '@material-ui/core/styles';
+import PropTypes from 'prop-types';
 
-function Cart() {
-  return (<div />);
+export const styles = {
+  cartHeading: {
+    padding: '16px',
+  },
+};
+
+function Cart({ classes, sections }) {
+  const word = sections.length === 1 ? 'Class' : 'Classes';
```

Comment:
I like the clean touch for handling singular cases for the UX, though I think we can come up with a better variable name than `word`. `suffix`? `label`? `classSuffix`?

Reply:
This should be